python/hyperspace/hyperspace.py

```diff
@@ -0,0 +1,173 @@
+import sys
+
+from pyspark.sql import SparkSession, DataFrame
+from hyperspace import *
+from py4j.java_gateway import java_import
+
+
+class Hyperspace:
+    def __init__(self, spark):
+        """
+        Initializes hyperspace object.
+        :param spark: sparkSession object
+        :return: hyperspace object
+
+        >>> hyperspace = Hyperspace(spark)
+        """
+        self.sparkContext = spark.sparkContext
+        self.spark = spark
+        self.jvm = spark._jvm
+        self.hyperspace = self.jvm.com.microsoft.hyperspace.Hyperspace(spark._jsparkSession)
+
+    def __getJavaIndexConfig(self, index_config):
+        """
+        Constructs IndexConfig Java object from python wrapper IndexConfig object.
+        :param index_config: IndexConfig java object
+        :return: IndexConfig python object
+
+        >>> __getJavaIndexConfig(idx_config)
+        """
+        indexed_columns = self.__getScalaSeqFromList(index_config.indexedColumns)
+        included_columns = self.__getScalaSeqFromList(index_config.includedColumns)
+        _jindexConfig = self.jvm.com.microsoft.hyperspace.index.IndexConfig(
```

Comment:
`_jIndexConfig`?

Reply:
_jindexConfig is more in alignment with what pyspark is following. Example: _jsparkSession, _jsc and so on.
Eg. https://github.com/apache/spark/blob/master/python/pyspark/sql/session.py#L229